gallery/gallery/lib/demos/material/snackbar_demo.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:gallery/l10n/gallery_localizations.dart';
+
+// BEGIN snackbarsDemo
+
+class SnackbarsDemo extends StatelessWidget {
+  int _snackBarIndex = 1;
```

Comment:
Simplify the demo to just show a static message

Reply:
Sounds good.